sys/net/gnrc/routing/rpl/gnrc_rpl_control_messages.c

```diff
@@ -229,32 +288,34 @@ void gnrc_rpl_send_DIO(gnrc_rpl_instance_t *inst, ipv6_addr_t *destination)
 
 void gnrc_rpl_send_DIS(gnrc_rpl_instance_t *inst, ipv6_addr_t *destination)
 {
-    gnrc_pktsnip_t *pkt;
+    gnrc_pktsnip_t *pkt = NULL, *tmp;
     icmpv6_hdr_t *icmp;
     gnrc_rpl_dis_t *dis;
 
-    /* TODO: Currently the DIS is too small so that wireshark complains about an incorrect
-     * ethernet frame check sequence. In order to prevent this, 4 PAD1 options are added.
-     * This will be addressed in follow-up PRs */
-    uint8_t padding[] = {
-            0x01, 0x02, 0x00, 0x00
-    };
+    if ((pkt = _dis_solicited_opt_build(pkt, inst)) == NULL) {
```

Comment:
Probably, but we still need to PadN 2 bytes to trick wireshark.
This way we handle the build in a separate function rather then `ifdef`ing even more here, 
which is something I would like to prevent. 

When we kick the PadN, I'm in.